pkg/krmtotf/tftokrm_test.go

```diff
@@ -2138,7 +2201,58 @@ func TestResolveSpecAndStatusContainingObservedState(t *testing.T) {
 			},
 		},
 		{
-			name: "with partial string observed fields in observed state",
+			name: "nested observed field not exist in the returned state but " +
```

Comment:
I think I understand the test case, but can the name be more descriptive? The parent does exist, not because it must exist, but because the parent has another child field, which has  an actual value being returned by the underlying API.

Reply:
Sorry for the confusion! What about the new name?